content/en/user-guide/aws/qldb/index.md

```diff
@@ -8,13 +8,309 @@ aliases:
+
+```bash
+qldb> SELECT r_id FROM VehicleRegistration AS r BY r_id
+WHERE r.VIN = 'KM8SRDHF6EU074761'
+{
+  r_id: "3TYR9BamzyqHWBjYOfHegE"
+}
+1 document in bag (read-ios: 0, server-time: 0ms, total-time: 541ms)
+```
+
+Then, the `id` is used to query the history function.
+
+```bash
+qldb> SELECT h.data.VIN, h.data.City, h.data.Owners
+FROM history(VehicleRegistration) AS h
+WHERE h.metadata.id = '3TYR9BamzyqHWBjYOfHegE'
+{
+  VIN: "KM8SRDHF6EU074761",
+  City: "Kent",
+  Owners: {
+    PrimaryOwner: {
+      PersonId: "294jJ3YUoH1IEEm8GSabOs"
+    },
+    SecondaryOwners: [
+      {
+        PersonId: "1nmeDdLo3AhGswBtyM1eYh"
+      },
+      {
+        PersonId: "IN7MvYtUjkp1GMZu0F6CG9"
+      }
+    ]
+  }
+},
+{
+  VIN: "KM8SRDHF6EU074761",
+  City: "Kent",
+  Owners: {
+    PrimaryOwner: {
+      PersonId: "112233445566NO"
+    },
+    SecondaryOwners: [
+      {
+        PersonId: "1nmeDdLo3AhGswBtyM1eYh"
+      },
+      {
+        PersonId: "IN7MvYtUjkp1GMZu0F6CG9"
+      }
+    ]
+  }
+}
+2 documents in bag (read-ios: 0, server-time: 0ms, total-time: 544ms)
+```
+
+### Cleaning up resources
+
+Unused ledgers can be deleted. You'll notice that directly running the following command will lead to an error message.
+
+{{< command >}}
+$ awslocal qldb delete-ledger --name vehicle-registration
+{{< / command >}}
+
+
+```bash
+An error occurred (ResourcePreconditionNotMetException) when calling the DeleteLedger operation: Preventing deletion 
+of ledger vehicle-registration with DeletionProtection enabled
+```
+
+This can be adjusted using the `update-ledger` command in the AWS CLI:
+
+{{< command >}}
+$ awslocal qldb update-ledger --name vehicle-registration --no-deletion-protection
+{
+  "Name": "vehicle-registration",
+  "Arn": "arn:aws:qldb:us-east-1:000000000000:ledger/vehicle-registration",
+  "State": "ACTIVE",
+  "CreationDateTime": 1697038061.0,
+  "DeletionProtection": false
+}
+{{< / command >}}
+
+Now the `delete-ledger` command can be repeated without errors.
+
+### Exploring further
+
+Interacting with Amazon QLDB (Quantum Ledger Database) is typically done using language-specific software 
+development kits (SDKs) provided by AWS. These SDKs make it easier for developers to interact with QLDB and 
+perform operations such as managing ledgers, executing PartiQL queries, and processing the results.
+When interacting with QLDB, it's common to use a combination of SDKs and PartiQL queries to achieve specific data 
+processing tasks, ensuring flexibility and ease of development.
+
+A simple QLDB example running on LocalStack is provided in [this Github repository](https://github.com/localstack/localstack-pro-samples/tree/master/qldb-ledger-queries).
+The sample consists of two simple scenarios:
+1. to create and list tables via the `pyqldb` Python library
+2. to insert data into two tables and perform a `JOIN` query that combines data from the two tables.
+
+The sample will output:
```

Comment:
Just add a hyperlink to the example with one-liner description

Reply:
i left only a brief description, as it's missing from the repo